src/lib/inline-images/InlineImages.tsx

```diff
@@ -61,12 +64,16 @@ const InlineImages = (props: InlineImagesPropType): JSX.Element => {
             left: `${data?.length * spaceBetweenPics}px`,
             width: `${size}px`,
             height: `${size}px`,
+            fontSize: `${
+              size *
+              (0.4 - 0.03 * Math.log10(Math.abs(totalUserCount - data?.length)))
```

Comment:
Do we need to add a comment to explain what this computation is intending to do ?